test/Microsoft.NET.Sdk.Functions.Generator.Tests/FunctionJsonConverterTests.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using FluentAssertions;
+using MakeFunctionJson;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.ServiceBus;
+using Microsoft.ServiceBus.Messaging;
+using Microsoft.WindowsAzure.Storage.Queue;
+using Xunit;
+
+namespace Microsoft.NET.Sdk.Functions.Test
+{
+    public class FunctionJsonConverterTests
+    {
+        public class FunctionsClass
+        {
+            [FunctionName("MyHttpTrigger")]
+            public static void Run1([HttpTrigger] HttpRequestMessage request) { }
+            
+            [FunctionName("MyBlobTrigger")]
+            public static void Run2([BlobTrigger("blob.txt")] string blobContent) { }
+            
+            [FunctionName("MyQueueTrigger")]
+            public static void Run3([QueueTrigger("queue")] CloudQueue queue) { }
+            
+            [FunctionName("MyEventHubTrigger")]
+            public static void Run4([EventHubTrigger("hub")] EventData message) { }
+            
+            [FunctionName("MyTimerTrigger")]
+            public static void Run5([TimerTrigger("00:30:00")] TimerInfo timer) { }
+            
+            [FunctionName("MyServiceBusTrigger")]
+            public static void Run6([ServiceBusTrigger("queue")] string message) { }
+                        
+            [FunctionName("MyManualTrigger"), NoAutomaticTrigger]
+            public static void Run7(string input) { }
```

Comment:
>input [](start = 43, length = 5)

Does this assume that the [ManualTrigger] is on the input parameter?

Reply:
Yes, see https://github.com/0xced/azure-functions-vs-build-sdk/blob/dce6e369444f6edb081fb09e22f0ed53fc82a643/src/Microsoft.NET.Sdk.Functions.Generator/MethodInfoExtensions.cs#L45

Reply:
Did you mean [NoAutomaticTrigger] instead of [ManualTrigger] ?